qiskit_experiments/database_service/json.py

```diff
@@ -19,29 +19,33 @@
 import numpy as np
 
 
-class NumpyEncoder(json.JSONEncoder):
+class ExperimentEncoder(json.JSONEncoder):
     """JSON Encoder for Numpy arrays and complex numbers."""
 
     def default(self, obj: Any) -> Any:  # pylint: disable=arguments-differ
         if hasattr(obj, "tolist"):
-            return {"type": "array", "value": obj.tolist()}
+            return {"__type__": "array", "__value__": obj.tolist()}
```

Comment:
Future discussion: should we use numpy serialization format for arrays since it is much faster and more efficient? The downside would be you probably can't then view them in IQX webpage.